### DISCUSSION TITLE
[Issue]: Radeon PRO W6800 Pro Graphics native Windows Pytorch acceleration support via ROCm / HIP or some other way?

### DISCUSSION THREAD

**Discussion by AshleyT3** [ROCm/ROCm#3794] [Q&A] [answered]
### Problem Description

Is there a plan to eventually support Radeon PRO W6800 Pro Graphics native Windows Pytorch acceleration support via ROCm / HIP or some other way? Today, it seems Pytorch is not supported natively on Windows aside from using DirectML. DirectML is usable but in at least one case it requires customizations to pre-existing Pytorch code so does not insert GPU acceleration support seamlessly in such a case.

I've seen some online commentary about ROCm 6.1 possibly supporting native Windows ROCm / Pytorch support but it's not official AMD commentary. If there is no official public word on this, it would be great to get some FAQ or comment on this. 

### Operating System

Windows 11

### CPU

5800X

### GPU

AMD Radeon Pro W6800

### ROCm Version

ROCm 6.0.0, ROCm 5.7.1

**Comment by ppanchad-amd** [comment]
@AshleyT3 Can you please close this ticket and open it under 'Discussions' section? Thanks!

**Comment by schung-amd** [comment, marked as answer]
Hi @AshleyT3, sorry for the lack of response. We're aware of the need for native Pytorch support on Windows for AMD GPUs and are working on it; stay tuned for formal announcements from AMD in the future. Thanks for your interest!
